velox/vector/tests/VectorTest.cpp

```diff
@@ -3899,5 +3899,21 @@ TEST_F(VectorTest, testOverSizedArray) {
   EXPECT_THROW(BaseVector::flattenVector(constArray), VeloxUserError);
 }
 
+TEST_F(VectorTest, testSliceBuffer) {
+  auto bufferPtr = AlignedBuffer::allocate<int64_t>(10, pool());
+  auto sliceBufferPtr =
+      BaseVector::sliceBuffer(*BIGINT(), bufferPtr, 1, 5, pool());
```

Comment:
What if we pass ARRAY(DOUBLE()) as type?

Reply:
This function would throw `std::invalid_argument{"Not a fixed width type: " + toString()}`.